### PR TITLE
luci-base: Update Simplified Chinese translation

### PR DESCRIPTION
**Pull request by daofeng2015**
Fixed Chinese translation error

### CODE REVIEW

**Comment by cshoredaniel** [comment]
@daofeng2015 Hi.  You efforts are appreciated, but could I trouble you to update the commit message to indicate what part of LuCI for which you are updating strings; we'd like to avoid non-specific commit messages.  Title is fine, but the commit message should be a little more specific.  Just saying something like 'updates strings in luci-base' is fine.  Hope you don't mind!

File: modules/luci-base/po/zh-cn/base.po [hunk 1]
```diff
@@ -1463,7 +1463,7 @@ msgid "Inactivity timeout"
 msgstr "活动超时"
 
 msgid "Inbound:"
-msgstr "入站："
+msgstr "上行："
```
Comment (cqhtyi) [inline modules/luci-base/po/zh-cn/base.po]: 不同意改为“上行”和“下行”，“出站”和“入站”更好理解。

File: modules/luci-base/po/zh-cn/base.po [hunk 2]
```diff
@@ -1289,13 +1289,13 @@ msgid "Host-<abbr title=\"Internet Protocol Address\">IP</abbr> or Network"
...
 msgid "Hostname"
-msgstr "主机名"
+msgstr "设备名"
 
 msgid "Hostname to send when requesting DHCP"
 msgstr "请求DHCP时发送的主机名"
 
 msgid "Hostnames"
-msgstr "主机名"
+msgstr "设备名"
```
Comment (cqhtyi) [inline modules/luci-base/po/zh-cn/base.po]: 不同意改为"设备名"，“主机名”这个翻译没有问题。

**Comment by hnyman** [comment]
no response from the PR opener, negative feedback.
closing.

**Pull request closed without merging**